capsules/extra/src/tickv.rs

```diff
@@ -45,7 +45,7 @@ use kernel::utilities::leasable_buffer::LeasableMutableBuffer;
 use kernel::ErrorCode;
 use tickv::{self, AsyncTicKV};
 
-#[derive(Clone, Copy, PartialEq)]
+#[derive(Clone, Copy, PartialEq, Debug)]
```

Comment:
Why do we need this to be `Debug`?

Reply:
I've done _a lot_ of debugging.

Reply:
Ha!

But it does add code size and it isn't being used by default

Reply:
I don't think it should add size in the case that it isn't being called